Sources/RichTextKit/Coordinator/RichTextCoordinator.swift

```diff
@@ -241,8 +241,9 @@ extension RichTextCoordinator {
             richTextContext.canUndoLatestChange = canUndo
         }
 
-        let fontName = textView.currentFontName ?? ""
-        if richTextContext.fontName != fontName {
+        if let fontName = textView.currentFontName,
```

Comment:
We dont want to set empty font name for sure... we should avoid this at any cost.